README.md

```diff
@@ -213,6 +214,30 @@ Advanced scripts with different models are available in the [mteb/mtebscripts re
 4. Hit the Refresh button at the bottom of the leaderboard and you should see your scores 🥇
 5. To have the scores appear without refreshing, you can open an issue on the [Community Tab of the LB](https://huggingface.co/spaces/mteb/leaderboard/discussions) and someone will restart the space to cache your average scores. The cache is updated anyways ~1x/week.
 
+
+
+## Contributing to MTEB
+We welcome contributions such as new datasets to MTEB! This section describes how to set up the repository for development.
+
+### Development Installation
+If you want to submit a dataset or on other ways contribute to MTEB, you can install the package in development mode:
+
+```bash
+git clone https://github.com/embeddings-benchmark/mteb
+pip install -e .
+pip install -r requirement.dev.txt
```

Comment:
along with `pip install -re requirements.txt` because it contains the packages needed to run mteb no?

Reply:
as least when I tested it it was not required.